src/flake8_import_conventions/checks.rs

```diff
@@ -0,0 +1,37 @@
+use rustc_hash::FxHashMap;
+use rustpython_ast::Stmt;
+
+use crate::ast::types::Range;
+use crate::checks::{Check, CheckKind};
+
+/// ICO001
```

Comment:
Do you prefer `ICO` or `ICN`? I see `ICN` in the enums but `ICO` here :)

Reply:
I ended up preferring `ICN` to avoid ambiguity in some fonts that might make `0` and `O` hard to tell apart